Userland/Libraries/LibSQL/BTree.h

```diff
@@ -6,6 +6,7 @@
 
 #pragma once
 
+#include "AK/Debug.h"
```

Comment:
Small style consistency thing:

```suggestion
#include <AK/Debug.h>
```